ivf/src/lib.rs

```diff
@@ -30,3 +30,54 @@ pub fn write_ivf_frame(
   bw.write(64, pts).unwrap();
   bw.write_bytes(data).unwrap();
 }
+
+#[derive(Debug, PartialEq)]
+pub struct Header {
+  pub tag: [u8; 4],
+  pub w: u16,
+  pub h: u16,
+  pub timebase_num: u32,
+  pub timebase_den: u32
+}
+
+pub fn read_header(r: &mut dyn io::Read) -> io::Result<Header> {
+  let mut br = BitReader::endian(r, LittleEndian);
+
+  let mut _signature = [0u8; 4];
+  let mut tag = [0u8; 4];
+
+  br.read_bytes(&mut _signature)?;
```

Comment:
Do you want to actually verify the signature?

Reply:
Probably it is a good idea.